packages/jest-snapshot/package.json

```diff
@@ -14,6 +14,7 @@
     "./package.json": "./package.json"
   },
   "dependencies": {
+    "@babel/core": "^7.7.2",
```

Comment:
I remember `optionalDependencies` should work.

Reply:
Not sure why it should be optional tbh

Reply:
Indeed Babel core is required for those Babel plugins